firmware/boron-ins-fsm/clang-format-check.py

```diff
@@ -0,0 +1,18 @@
+#
+# clang-format-check.py
+#
+# Checks whether all files with filetype .cpp or .h in the src and test directories are formatted correctly. 
+# The syntax of this script has been modified to work in Travis CI. In order to run it locally, change all references to clang-format-12 to clang-format. 
```

Comment:
You could probably also rename the executable on your local machine and not need to do it in this script. But thank you for committing the version that Travis needs to run.

Reply:
This is true. No problem